src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$calendar-day-color: #eee;
+$calendar-lenght: 31;
```

Comment:
There seems to be a typo in the variable name `$calendar-lenght`. The correct spelling is `length`. This could lead to misunderstandings or errors when the variable is used elsewhere in the code.